packages/design-system/src/components/InlineError/InlineError.tsx

```diff
@@ -9,24 +11,31 @@ import classNames from 'classnames';
 interface InlineErrorProps {
   children?: React.ReactNode;
   className?: string;
+  displayErrorIcon?: boolean;
   id?: string;
   inversed?: boolean;
 }
 
 export function InlineError({
   children,
   className,
+  displayErrorIcon,
   id,
   inversed,
 }: InlineErrorProps): React.ReactElement {
+  getInlineErrorIconDisplay() ? (displayErrorIcon = true) : (displayErrorIcon = false);
+
   const classes = classNames(
+    'ds-c-inline-error',
     'ds-c-field__error-message',
     { 'ds-c-field__error-message--inverse': inversed },
     className
   );
+  const viewbox = '36 -6 186 186';
```

Comment:
This seems to have done nearly a perfect job of matching the old one. Looks like we're maybe 1 pixel offset between https://cmsgov.github.io/design-system/healthcare/example/components.form-label.react/ and http://localhost:3000/example/components.form-label.react/

Reply:
I like the new spacing better